RockLib.UniversalMemberAccessor/UniversalMemberAccessor.cs

```diff
@@ -68,40 +71,40 @@ static UniversalMemberAccessor()
 
             try
             {
-                staticValueTypeField.SetValue(null, 9);
+                staticValueTypeField?.SetValue(null, 9);
                 _canSetReadonlyStaticValueType = (ReadonlyFields.StaticValueType != initialStaticValueType);
             }
-            catch
+            catch (Exception ex) when (ex is FieldAccessException || ex is TargetException || ex is ArgumentException)
```

Comment:
I found this funny, because this basically allows you to catch specific exceptions that all do the same thing with one statement. That's kind of a nice trick :)